data/datacatalogue/CatalogueOntologies/Study design classification.csv

```diff
@@ -1,29 +1,29 @@
-name,definition,parent
-Clinical Trial Randomisation,,
-Randomised clinical trial,,Clinical Trial Randomisation
-Non-randomised clinical trial,,Clinical Trial Randomisation
-Clinical Trial Types,,
-Low-interventional clinical trial,,Clinical Trial Types
-Single-arm trial,,Clinical Trial Types
-Large simple trial,,Clinical Trial Types
-Pragmatic clinical trial,,Clinical Trial Types
-Cluster randomised trial,,Clinical Trial Types
-Non-interventional design,,
-Cohort,,Non-interventional design
-Case-control,,Non-interventional design
-Case-only,,Non-interventional design
-Cross-sectional,,Non-interventional design
-Ecological,,Non-interventional design
-Cluster design,,Non-interventional design
-Systematic review and meta-analysis,,Non-interventional design
-Other,,Non-interventional design
-Clinical trial phase,,
-phase 1,,Clinical trial phase
```

Comment:
phase 1 to phase 4 renamed to: 
Human pharmacology (Phase I)
Therapeutic exploratory (Phase II)
Therapeutic confirmatory (Phase III)
Therapeutic use (Phase IV)

Reply:
was this requested by EMA?

Reply:
I see that RMS was also changed, so ok.